test/e2e/network/netpol/test_helper.go

```diff
@@ -68,23 +69,25 @@ func UpdatePolicy(k8s *kubeManager, policy *networkingv1.NetworkPolicy, namespac
 func ValidateOrFail(k8s *kubeManager, model *Model, testCase *TestCase) {
 	ginkgo.By("Validating reachability matrix...")
 
-	// 1st try
-	ginkgo.By("Validating reachability matrix... (FIRST TRY)")
-	ProbePodToPodConnectivity(k8s, model, testCase)
-	// 2nd try, in case first one failed
-	if _, wrong, _, _ := testCase.Reachability.Summary(ignoreLoopback); wrong != 0 {
-		framework.Logf("failed first probe %d wrong results ... retrying (SECOND TRY)", wrong)
+	err := wait.PollImmediate(3*time.Second, 15*time.Second, func() (done bool, err error) {
```

Comment:
How did you arrive at 3 seconds for interval, and 15 seconds for timeout?  

Looking at [a recent test run](https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-ubuntu-gce-network-policies/1409703242257928192/build-log.txt), a `ProbePodToPodConnectivity` probe can take ~ 2 minutes.  With the timeout set to 15 seconds, wouldn't this change effectively stop any retries from happening?  I'm not sure that's what we want.

Reply:
An educated guess from @danwinship. I'll take a closer look, thanks!